gnomad_qc/v5/sample_qc/hard_filters.py

```diff
@@ -0,0 +1,115 @@
+"""Script to determine samples that fail hard filtering thresholds."""
+
+import argparse
+import logging
+
+import hail as hl
+from gnomad.resources.grch38.reference_data import telomeres_and_centromeres
+from gnomad.sample_qc.filtering import compute_stratified_sample_qc
+from gnomad.utils.annotations import bi_allelic_expr
+
+from gnomad_qc.v5.resources.basics import get_aou_vds
+from gnomad_qc.v5.resources.sample_qc import get_aou_sample_qc
+
+logging.basicConfig(format="%(levelname)s (%(name)s %(lineno)s): %(message)s")
+logger = logging.getLogger("hard_filters")
+logger.setLevel(logging.INFO)
+
+
+def compute_aou_sample_qc(
+    n_partitions: int = 500,
+    test: bool = False,
+) -> hl.Table:
+    """
+    Perform sample QC on the VDS.
+
+    :param n_partitions: Number of partitions to use when writing the sample QC table.
+    :param test: If true, test the function on a smaller subset of the data.
+    :return: Table containing sample QC metrics
+    :rtype: hl.Table
+    """
+    logger.info("Computing sample QC")
+
+    if test:
+        logger.info("Loading test VDS...")
+        vds = get_aou_vds(
+            test=True,
+            autosomes_only=True,
+            split=True,
+        )
+        n_partitions = n_partitions // 100
+    else:
+        logger.info("Loading full VDS...")
+        vds = get_aou_vds(
+            autosomes_only=True,
+            split=True,
+        )
```

Comment:
this can just call the function once and pass in the test arg right?

vds = get_aou_vds(
            test=test,
            autosomes_only=True,
            split=True,
        )